src/index.html

```diff
@@ -9,10 +9,56 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/styles.scss"
+    />
```

Comment:
The stylesheet link references a SCSS file (`styles.scss`). SCSS files need to be compiled into CSS before they can be used in an HTML document. Ensure that the correct CSS file is linked here.